samples/core/exit_handler/exit_handler.py

```diff
@@ -13,37 +13,35 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-
 import kfp
-from kfp import dsl
-
-
-def gcs_download_op(url):
-    return dsl.ContainerOp(
-        name='GCS - Download',
-        image='google/cloud-sdk:279.0.0',
-        command=['sh', '-c'],
-        arguments=['gsutil cat $0 | tee $1', url, '/tmp/results.txt'],
-        file_outputs={
-            'data': '/tmp/results.txt',
-        }
+import os
+from kfp import dsl, components
+
+gcs_download_op = kfp.components.load_component_from_file(
```

Comment:
We could probably use `ComponentStore` configured with both local path and remote URL.

Reply:
Can you provide a sample?
I just realized local import might not be possible in legacy e2e tests https://oss-prow.knative.dev/view/gs/oss-prow/pr-logs/pull/kubeflow_pipelines/5473/kubeflow-pipeline-e2e-test/1382342847985881088#1:build-log.txt%3A5624.

I need to use an url import here.

Reply:
```
component_store = components.ComponentStore(
    local_search_paths=[
        '.',
    ],
    url_search_prefixes=[
        'https://raw.githubusercontent.com/kubeflow/pipelines/master/components/'
    ],
)
...
op = component_store.load_component(name='google-cloud/storage/download_blob')
```